.github/workflows/sims.yml

```
@@ -12,6 +12,14 @@ jobs:
   build:
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, 'skip-sims')"
+    steps:
+      - uses: actions/setup-go@v2-beta
+      - uses: actions/checkout@v2
+      - run: |
+          make build
+  runsim:
```

```suggestion
  install-runsim:
```

I think it better describes what the task actually does?